app/scaffold/scaffold.js

```diff
@@ -144,11 +144,25 @@ class Scaffold {
             const crewmemberIdsToAdd = DOM.getSelectedValues(DOM.fromElement);
 
             // if add has been implemented, reflect changes in model and view
-            if (this.addAwayTeamMembersFn(crewmemberIdsToAdd)) {
-                model.addAwayTeamMembers(crewmemberIdsToAdd);
-                DOM.moveItemsBetweenSelects(DOM.fromElement, DOM.toElement);
+            const addMembersPromise = this.addAwayTeamMembersFn(crewmemberIdsToAdd);
+            if(!addMembersPromise){
+                return;
+            }
+
+            if(crewmemberIdsToAdd.length === 0){
+                DOM.alertBox("Please select a crew member to add");
+            }else{
```

Comment:
Nitpick that is failing `.eslint`: 1) there should be a space after the brace, 2) the `else` should be on a new line.

Also, try to remember to run the linter before pushing up.